.github/workflows/pr.yaml

```diff
@@ -168,16 +167,16 @@ jobs:
       - name: Buld tracee-tester image
         run: |
           make -f tests/Makefile docker-build-tracee-tester
-      - name: Build tracee-nocore
+      - name: Build tracee container image (full w/out BTFHUB)
```

Comment:
Full is the old "fat" image, without BTFHUB support means it has to build it during its initialization and place in /tmp/tracee (unless it already exists there, then no build is needed).